lib/internal/per_context/setup.js

```diff
@@ -1,20 +1,19 @@
 // This file is compiled as if it's wrapped in a function with arguments
 // passed by node::NewContext()
-/* global global */
 
 'use strict';
 
 // https://github.com/nodejs/node/issues/14909
-if (global.Intl) delete global.Intl.v8BreakIterator;
+if (Intl) delete Intl.v8BreakIterator;
```

Comment:
Doesn't this throw a ReferenceError if Intl is not defined?

Reply:
Thanks, fixed.